### DISCUSSION TITLE
Errors When Querying JSON With DBeaver

### DISCUSSION THREAD

**Discussion by najmead** [duckdb/duckdb#4181] [Q&A]
I've been using duckdb's json functions to parse some text, and have been consistently getting errors.  After experimenting a little, it looks like the issue is with dbeaver because the queries work ok using duckdb's CLI.

eg, 

`select json('{}')
`

Gives the error
`
SQL Error: Internal jdbc driver error
`

But works fine from the CLI.

I'm using v0.4 of the CLI and v0.4 of the JDBC driver in DBeaver.  Is there something I'm missing?

**Comment by wainage** [comment]
Using DataGrip I can see JDBC error `No enum constant org.duckdb.DuckDBColumnType.JSON`.

The [DuckDBColumnType.java](https://github.com/duckdb/duckdb/blob/463e45ba98754b0e2fdc8bc5b1481dfbc7a7398d/tools/jdbc/src/main/java/org/duckdb/DuckDBColumnType.java) file is missing the JSON enum.

**Comment by lnkuiper** [comment]
As a workaround for now, you can cast the JSON to VARCHAR.
